src/Common/Common.Api/Datasets/Dataset.cs

```diff
@@ -51,6 +51,21 @@ public class Dataset
             };
         }
 
+        public static PowerBI.Api.V2.Models.Dataset ConvertToPushDataset(Dataset dataset)
```

Comment:
Change name to ConvertToDatasetV2Model

Reply:
Or can we switch this to be an operator?

Reply:
Yes, it was operator and i couldn't map all properties. Then I changed it to this. At the moment, PushDataset is the only one using this conversion. Could you check the previous commit to see if it was okay?